### DISCUSSION TITLE
video "Pytorch Fundamentals"-what is "shape" for a tensor?

### DISCUSSION THREAD

**Discussion by MALLIKARJUNABATHALA999** [mrdbourke/pytorch-deep-learning#790] [Q&A]
hello

**Comment by ZephyrusBlaze** [comment]
In PyTorch, the "shape" of a tensor refers to its dimensions or the size of each dimension. For example, a tensor with a shape of (3, 4) would have 3 rows and 4 columns if it's a 2-dimensional tensor. In general, the shape of a tensor describes its size along each axis or dimension.
